engine/message/db.go

```diff
@@ -277,6 +279,21 @@ func NewDB(ctx context.Context, db *sql.DB, c *Config, a alertlog.Store) (*DB, e
 			) select distinct msg_id, alert_id, user_id, cm_id from disabled
 		`),
 
+		failSMSVoice: p.P(`
+			update outgoing_messages msg
+			set
+				last_status = 'failed',
+				last_status_at = now(),
+				status_details = 'twilio disabled',
+				cycle_id = null,
+				next_retry_at = null
+			from user_contact_methods cm
+			where
+				msg.last_status = 'pending' and
+				cm.type in ('SMS', 'VOICE')
+			returning msg.id as msg_id, alert_id, msg.user_id, cm.id as cm_id
```

Comment:
We need to ensure we're comparing `.type` on the contact method associated with the message.
```suggestion
			where
				msg.last_status = 'pending' and
				cm.type in ('SMS', 'VOICE') and
				cm.id = msg.id
			returning msg.id as msg_id, alert_id, msg.user_id, cm.id as cm_id
```